packages/cojson/src/storage/storageAsync.ts

```diff
@@ -290,38 +320,31 @@ export class StorageApiAsync implements StorageAPI {
     storedCoValueRowID: number,
   ) {
     const newTransactions = msg.new[sessionID]?.newTransactions || [];
+    const lastIdx = sessionRow?.lastIdx || 0;
```

Comment:
This is just a light refactoring to make the code more readable and gain more understanding of it.